int32 kitchen_time
int32 lounge_time
int32 entrance_time
int32 lobby_time
int32 tvRoom_time
int32 bedRoom_time
int32 away_time